RELEASE_NOTES.rst

```diff
@@ -42,7 +42,7 @@ coala 0.8.0 - grizzly
 For this release, we have had 46 developers from around the world contributing
```

Comment:
@adtac I havent quite understood what you mean to say, Could you explain ? Should i revert back all "author" to "writer" in RELEASE_NOTES.rst ??